bluemira/equilibria/coils/_coil.py

```diff
@@ -154,8 +154,8 @@ def __init__(
 
         self.x = x
         self.z = z
-        self.dx = dx
-        self.dz = dz
+        self._dx = dx
+        self._dz = dz
```

Comment:
revert I think? the setters are needed otherwise this is just a repeat of L147/148

Reply:
Ah right, I was getting confused.